kite-data/kite-data-core/pom.xml

```diff
@@ -271,5 +271,36 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>hdp2-release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-deploy-plugin</artifactId>
+            <version>${vers.maven-deploy-plugin}</version>
+            <executions>
+              <execution>
+                <id>deploy-hdp-classifier</id>
```

Comment:
I'm guessing it's ok to duplicate the execution id since it's in two different profiles.